### DISCUSSION TITLE
Welcome to PyGameExamplesAndAnswers Discussions!

### DISCUSSION THREAD

**Discussion by Rabbid76** [Rabbid76/PyGameExamplesAndAnswers#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Daniel-Zheleznov** [comment]
Hey Raddib amazing work but may you help me out a bit. May you discuss how you would make a dropdown. (like the core concepts of it) The Method Behind the Madness, if you will?
